datalad_gooey/simplified_api.py

```diff
@@ -34,14 +34,14 @@
         ),
     ),
     create=dict(
-        name='C&reate a dataset',
+        name='Create a dataset',
```

Comment:
Please do not remove the keyboard navigation hint!

Reply:
Sorry! thought that was a typo, didn't see it in the others haha..